lib/hubspot/company.rb

```diff
@@ -165,6 +165,18 @@ def add_contact!(company_vid, contact_vid)
                                      },
                                      body: nil)
       end
+
+      # Updates the properties of a company
+      # {http://developers.hubspot.com/docs/methods/companies/update_company}
+      # @param vid [Integer] hubspot company vid
+      # @param params [Hash] hash of properties to update
+      # @return [Hubspot::Company] Company record
+      def update!(vid, params)
+        params.stringify_keys!
```

Comment:
Do we need to call `stringify_keys` here?

Reply:
Seems unnecessary as it looks like `Hubspot::Utils::hash_to_properties` calls `to_s` on the keys:
https://github.com/adimichele/hubspot-ruby/blob/945a8717f260fc064ef6b34f427fc3333f28aaf4/lib/hubspot/utils.rb#L20